http/handler.go

```diff
@@ -107,7 +107,6 @@ var (
 
 func init() {
 	alwaysRedirectPaths.AddPaths([]string{
-		"sys/storage/raft/snapshot",
```

Comment:
The path manager object we are adding these paths to has the ability to exempt a path by prefixing it with `!`. Instead of removing this path we should probably add the snapshot-auto path as an exemption. What do you think?

https://github.com/hashicorp/vault/blob/6a8e377b725562eb6b7689f5300ba87d64fbb455/sdk/helper/pathmanager/pathmanager.go#L35-L38